ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -99,8 +99,7 @@ QtObject {
     }
 
     function copyToClipboard(text) {
-        // Not Refactored Yet
-//        chatsModelInst.copyToClipboard(text);
+        globalUtils.copyToClipboard(text)
```

Comment:
what is globalUtils?

Reply:
It's the new Nim `Utils` View. https://github.com/status-im/status-desktop/blob/refactoring/base_bc/src/app/global/utils.nim